com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -505,7 +507,6 @@ static void RenderCameraStack(ScriptableRenderContext context, Camera baseCamera
             //It should be called before culling to prepare material. When there isn't any VisualEffect component, this method has no effect.
             VFX.VFXManager.PrepareCamera(baseCamera);
 #endif
-            UpdateVolumeFramework(baseCamera, baseCameraAdditionalData);
 #if ADAPTIVE_PERFORMANCE_2_0_0_OR_NEWER
```

Comment:
@PaulDemeulenaere for awareness. Is there any issue with calling UpdateVolumeFramework before VFXManager.PrepareCamera

Reply:
PrepareCamera will not access to any global settings, it will mainly update material, if camera isn't modified by UpdateVolumeFramework I don't expect any side effect.